src/systems/ec/design-tokens/tokens/colors.yml

```diff
@@ -26,6 +25,9 @@ aliases:
   COLOR_GREY_15: '#e3e3e3'
   COLOR_GREY_10: '#ebebeb'
   COLOR_GREY_5: '#f5f5f5'
+  COLOR_BLUE_N: '#006fb4'
   COLOR_RED_100: '#da2131'
```

Comment:
would be nice to rename `COLOR_RED_100` -> `COLOR_RED` to match the colour palette (same with `COLOR_GREEN_100` and `COLOR_ORANGE_100`)

the `100` can lead you to think there are other shades of red/green/orange, while it's not (yet) the case

what do you think?